init/initialize.sh

```diff
@@ -97,6 +97,15 @@ sed -e "s;RSYNCDIR_SS;$RSYNCDIR_SS;" \
     -e "s;RSYNCDIR_NPAD;$RSYNCDIR_NPAD;" \
     -e "s;RSYNCDIR_PTR;$RSYNCDIR_PTR;" \
     $SLICEHOME/conf/rsyncd.conf.in > /etc/rsyncd.conf
+
+# Allow CIRA rsync access to the paris-traceroute module at their own sites
+for site in "yyz01 yyc01 yul01"; do
+    if echo $(hostname) | grep -q $site; then
```

Comment:
May I suggest a bash-only approach? `$HOSTNAME` is defined in /etc/profile, so should be available here. And, I think the bash `=~` matching operator could check the hostname for matching sites. Does this work as intended?

    if [[ "$HOSTNAME" =~ "$site" ]] ; then
    ...
    fi